gateway/api.go

```diff
@@ -2835,6 +2835,8 @@ func (gw *Gateway) makeImportedOASTykAPI(next http.HandlerFunc) http.HandlerFunc
 			return
 		}
 
+		oasObj.GetTykExtension().Server.ListenPath.Strip = true
```

Comment:
I think this should be inside BuildDefaultTykExtension.

Reply:
We use buildDefaultTykExtension is also used in patch endpoint

Reply:
We can pass a param. Building the extension is that function’s job.

Reply:
“If post” check should work

Reply:
Actually, it'd make sense pass a param to handle some other flags as well.
Internal/external, active etc.

Will make those changes as well